<html>
<style>
#aboutBox img {
    height: 5em;
    margin-left: 1em;
    float: right;
    background-color: black;
}

#aboutBox ul {list-style-type: none}
    
#aboutBox ul img {
    border: 1px solid grey;
    width: 50px;
    height:50px;
    margin-right: 10px;
    margin-left: -60px;
    float: left;
    border-radius:50%;
    clear:both;
    
}

#aboutBox ul {
    margin:0;
    padding: 0;
    padding-left: 1em;
}

#aboutBox h2, li {
    clear:both;
}

#aboutBox li {
    margin: 0.5em 0;
    margin-left: 50px;
}
</style>
<head>
</head>
<body id="aboutBox">
<h2>List of data sources</h2>

<h3>Images</h3>

<p><img src="/static/FinalOutputs/pics/26793885.jpg" alt="" /> Images of organisms have all been taken from the <a href=""eol.org">Encylopedia of Life</a>, and are variously in the public domain, or can be reused under the <a href="">Creative Commons attribution-only</a> or <a href="">Creative Commons attribution + share-alike</a> licences. To see the source for any image, zoom in to the copyright symbol at the lower right, or click on the symbol to go to the the Encylopedia of Life page from where more information can be obtained.</p>

<h3>Evolutionary Trees</h3>

<p>The OneZoom tree of life is based on a bespoke <q>backbone</q>, originally focussed on vertebrate species, but now extended more widely to encompass all archaea and eukaryotes.</p>

<p><a href="http://www.opentreeoflife.org"><img src="/static/images/Open-Tree-of-Life-Logo.png" alt="" /></a>Species have been grafted onto this backbone using a large number of studies of various groups or organisms. In particular, many studies have been collected by the <a href="http://www.opentreeoflife.org">Open Tree of Life</a> project, an <a href="http://www.pnas.org/content/112/41/12764">ambitious attempt</a> to unify scientific studies into a comprehensive evolutionary tree of all known species of life on earth. This provides comprehensive coverage of approximately 2.1 million species, but the current draft version contains some problematic arrangements of relationships between higher-level groups (we encourage evolutionary scientists to <a href="https://tree.opentreeoflife.org/curator">add data to the OpenTree</a>, from where it will eventually percolate down into the OneZoom software). </p>

<p>For added accuracy, and to provide a few divergence dates, the OneZoom site relies on the additional studies listed below for specific areas of the tree of life. Where possible, the OpenTree (<a href="http://files.opentreeoflife.org/synthesis/opentree7.0">version 7</a>)  has been used to supplement these studies to include most or all currently recognized species. Note that the OneZoom display currently requires all branches to split in two (bifurcations), rather than simultanously splitting into 3 or more branches (technically known as polytomies). Where polytomies exist in the trees used, the OneZoom software splits them randomly into bifurcations.</p>

<ul>
<li><img src="/static/FinalOutputs/pics/26793963.jpg" alt="" /><strong>Mammals</strong>: the entire tree is from Bininda-Emonds <i>et al.</i> (2007) <i>The delayed rise of present-day mammals</i> (<a href="http://www.nature.com/nature/journal/v446/n7135/full/nature05634.html">Nature <b>446</b>: 507-512</a>), with the exception of:

<ul>
<li><img src="/static/FinalOutputs/pics/26811783.jpg" alt="" /><strong>Marsupials</strong>: orders arranged according to Nilsson <i>et al.</i> (2010) <i>Tracking Marsupial Evolution Using Archaic Genomic Retroposon Insertions</i> (<a href="http://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1000436">PLoS Biology: <b>8</b>: e1000436</a>).</li>
<li><img src="/static/FinalOutputs/pics/26808924.jpg" alt="" /><strong>Placental mammals</strong>: afrotheres and xenarthrans are grouped together (the <q>atlantogenata</q> hypothesis) on the basis of Poulakakis &amp; Stamatakis (2010) <i>Recapitulating the evolution of Afrotheria: 57 genes and rare genomic changes (RGCs) consolidate their history</i> (<a href="http://dx.doi.org/10.1080/14772000.2010.484436">Systematics and Biodiversity: <b>8</b> 395-408</a>).</li>
<li><img src="/static/FinalOutputs/pics/26836324.jpg" alt="" /><strong>Cetartiodactyla</strong>: placement is taken from dos Reis <i>et al.</i> (2012) <i>Phylogenomic datasets provide both precision and accuracy in estimating the timescale of placental mammal phylogeny</i> (<a href="http://rspb.royalsocietypublishing.org/content/279/1742/3491.long">Proceedings of the Royal Society B: Biological Sciences <b>279</b>: 3491-3500</a>).</li>
<li><img src="/static/FinalOutputs/pics/26805785.jpg" alt="" /><strong>Colugos</strong> from Janeƒçka <i>et al.</i> (2008) <i>Evidence for multiple species of Sunda colugo</i> (<a href="http://www.sciencedirect.com/science/article/pii/S0960982208011901">Current Biology <b>18</b>: R1001-R1002</a>).</li>
<li><img src="/static/FinalOutputs/pics/31340940.jpg" alt="" /><strong>Primates</strong>: entire tree from Springer <i>et al.</i> (2012) <i>Macroevolutionary dynamics and historical biogeography of primate diversification inferred from a species supermatrix</i> (<a href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0049521">PLoS ONE <b>7</b>: e49521</a> supplementary information text s2), with the exception of:

<ul>
<li><img src="/static/FinalOutputs/pics/26838690.jpg" alt="" /><b>Gibbons</b>: arrangements of the 4 genera taken from Carbone <i>et al.</i> (2014) <i>Gibbon genome and the fast karyotype evolution of small apes</i> (<a href="http://www.nature.com/nature/journal/v513/n7517/full/nature13679.html">Nature <b>513</b>: 195-201</a>).</li>
</ul></li>
</ul></li>
<li><img src="/static/FinalOutputs/pics/13168984.jpg" alt="" /><strong>Sauropsids</strong> (lizards, snakes, crocodiles, birds, turtles):
<ul>
<li><img src="/static/FinalOutputs/pics/31322029.jpg" alt="" /><strong>Birds</strong>: entire tree from Jetz <i>et al.</i> (2012) The global diversity of birds in space and time</i> (<a href="http://www.nature.com/nature/journal/v491/n7424/full/nature11631.html">Nature <b>491</b>: 444-448</a>). See also <a href="http://www.birdtree.org">www.birdtree.org</a>.</li>
<li><img src="/static/FinalOutputs/pics/23313777.jpg" alt="" /><strong>Snakes &amp; lizards</strong>: arrangement and divergence dates of Gekkota, Xantusiidae, Gerrhosauridae, Cordylidae, Scincidae, Lacertoidea, Anguimorpha, Acrodonta, Pleurodonta, Serpentes, and species-level taxonomy of Sphenodon and Dibamus from Zheng &amp; Wiens (2016) <i>Combining phylogenomic and supermatrix approaches, and a time-calibrated phylogeny for squamate reptiles (lizards and snakes) based on 52 genes and 4162 species</i> (<a href="http://www.sciencedirect.com/science/article/pii/S1055790315003127">Molecular Phylogenetics and Evolution <b>94</b>: 537-547</a>). Lower-level taxonomy is taken from the OpenTree.</li>
<li><img src="/static/FinalOutputs/pics/13169012.jpg" alt="" /><strong>Turtles</strong>: entire tree from Jaffe <i>et al.</i> (2011) <i>The evolution of island gigantism and body size variation in tortoises and turtles</i> (<a href="http://rsbl.royalsocietypublishing.org/content/7/4/558">Biology Letters <b>7</b>: 558-561</a>)</li>
<li><img src="/static/FinalOutputs/pics/22741720.jpg" alt="" /><b>Crocodiles</b>: entire tree from Oaks (2011) <i>A time-calibrated species tree of crocodylia reveals a recent radiation of the true crocodiles</i> (<a href="http://www.ncbi.nlm.nih.gov/pubmed/22023592">Evolution <b>65</b>: 3285-97</a>)</li>
</ul></li>
<li><img src="/static/FinalOutputs/pics/26929838.jpg" alt="" /><p><b>Lungfish</b>: entire phylogeny and dates from Criswell (2015) <i>The comparative osteology and phylogenetic relationships of African and South American lungfishes (Sarcoptergii:
Dipnoi)</i> (<a href="">Zoological Journal of the Linnean Society 174: 801-858</a>).</p></li>
<li> <img src="/static/FinalOutputs/pics/26919672.jpg" alt="" /> <strong>Ray-finned fish</strong>: backbone from <a href="http://currents.plos.org/treeoflife/article/the-tree-of-life-and-a-new-classification-of-bony-fishes/">DeepFin (version 3)</a> with the Open Tree of Life used to fill out the following orders and groups to species level: Elopiformes, Albuliformes, Notacanthiformes, Anguilliformes, Hiodontiformes, Osteoglossiformes, Clupeiformes, Alepocephaliformes, Gonorynchiformes, Cypriniformes, Gymnotiformes, Citharinoidei, Characoidei, Siluriformes, Argentiniformes, Galaxiiformes, Esociformes, Salmoniformes, Osmeriformes, Stomiiformes, Ateleopodiformes, Aulopiformes, Myctophiformes, Lampriformes, Percopsiformes, Zeiformes, Gadiformes, Polymixiiformes, Beryciformes plus Holocentriformes, Ophidiimorpharia, Batrachoidaria (Batrachoidimorpharia), Gobiomorpharia, Syngnathidae, Solenostomidae, Dactylopteroidei,Pegasidae,Callionymoidei, Pelagimorpharia, Synbranchiformes, Anabantiformes, Carangimorphariae, Ovalentaria, and Percomorpharia. For other groups, the following references have been used:
<ul>
<li><img src="/static/FinalOutputs/pics/2005176.jpg" alt="" /><strong>Sturgeons</strong>: entire tree from Krieger <i>et al.</i> (2008) <i>The molecular phylogeny of the order Acipenseriformes revisited</i> (<a href="http://onlinelibrary.wiley.com/doi/10.1111/j.1439-0426.2008.01088.x/abstract">Journal of Applied Ichthyology <b>24</b> Supplement s1: 36-45</a>).</li>
<li><img src="/static/FinalOutputs/pics/31401638.jpg" alt="" /><strong>Gars</strong>: all species from Deepfin 3, with <i>Atractosteus tristoechus</i> added from from Wright <i>et al.</i> (2012) <i>Gene trees, species trees, and morphology converge on a similar phylogeny of living gars (Actinopterygii: Holostei: Lepisosteidae), an ancient clade of ray-finned fishes</i> (<a href="http://dx.doi.org/10.1016/j.ympev.2012.02.033">Molecular Phylogenetics and Evolution <b>63</b>: 848-856</a>).</li>
<li><img src="/static/FinalOutputs/pics/27043111.jpg" alt="" /><strong>Bichirs</strong> from Suzuki <i>et al.</i> (2010) <i>The mitochondrial phylogeny of an ancient lineage of ray-finned fishes (Polypteridae) with implications for the evolution of body elongation, pelvic fin loss, and craniofacial morphology in Osteichthyes</i> (<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3055249/">BMC Evolutionary Biology <b>10</b>: 209</a>).</li>
</ul></li>
<li><img src="/static/FinalOutputs/pics/26822090.jpg" alt="" /><strong>Sharks, Rays, &amp; Chimeras</strong> (cartilagenous fish): see the <a href="http://sharksrays.org">Chondrichthyan Tree of Life</a>. Timing of the basal split is from Renz <i>et al.</i> (2013) <i>Revealing Less Derived Nature of Cartilaginous Fish Genomes with Their Evolutionary Time Scale Inferred with Nuclear Genes</i> (<a href="http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0066400">PLoS ONE <b>8</b>: e66400</a>), and species-level detail comes from the OpenTree of Life, with higher-level groupings as follows:
<ul>
<li><img src="/static/FinalOutputs/pics/29961780.jpg" alt="" /><strong>Chimaeras</strong>: arrangement of 6 major groups from Inoue <i>et al.</i> (2010) <i>Evolutionary Origin and Phylogeny of the Modern Holocephalans (Chondrichthyes: Chimaeriformes): A Mitogenomic Perspective</i> (<a href="http://mbe.oxfordjournals.org/content/27/11/2576.abstract">Molecular Biology and Evolution <b>27</b>: 2576-2586</a>).</li>
<li><img src="/static/FinalOutputs/pics/26792737.jpg" alt="" /><strong>Skates &amp; Rays</strong>: arrangement of 16 higher-level groups from Aschliman et al. (2012) <i>Body plan convergence in the evolution of skates and rays (Chondrichthyes: Batoidea)</i> 
(<a href="http://www.sciencedirect.com/science/article/pii/S1055790311005252">Molecular Phylogenetics and Evolution <b>63</b>: 28-42</a>).</li>
<li><img src="/static/FinalOutputs/pics/26814813.jpg" alt="" /><strong>Sharks</strong>: relationships between genera (also within genera for Etmopteridae, Pristiophoridae, and <i>Atelomycterus</i>) primarily based on Naylor <i>et al.</i> (2012) <i>Elasmobranch Phylogeny: A Mitochondrial Estimate Based on 595 Species</i> (<a href="https://books.google.co.uk/books?id=7qXMBQAAQBAJ&amp;pg=PA31">In: Carrier, Musick, &amp; Heithaus (eds) Biology of Sharks and their Relatives, Edition 2. CRC Press, Boca Raton, Florida: 31-56</a>), apart from:

<ul>
<li><img src="/static/FinalOutputs/pics/27186039.jpg" alt="" /><strong>Squaliform</strong> (deep sea) shark phylogeny from Straube <i>et al.</i> (2015) <i>Molecular phylogeny of Squaliformes and first occurrence of bioluminescence in sharks</i> (<a href="http://www.biomedcentral.com/1471-2148/15/162">BMC Evolutionary Biology <b>15</b>: 162</a>).</li>
<li><i>Centroscymnus</i> &amp; <i>Proscymnodon</i>: these genera are currently excluded.</li>
</ul></li>
</ul></li>
<li><img src="/static/FinalOutputs/pics/26779970.jpg" alt="" /><p><strong>Cyclostomes</strong></p>
<ul>
<li><img src="/static/FinalOutputs/pics/27280570.jpg" alt="" /><strong>Hagfishes</strong>: arrangement of genera from a combination of the 3 trees in Fernholm <i>et al.</i> (2013) <i>Hagfish phylogeny and taxonomy, with description of the new genus Rubicundus (Craniata, Myxinidae)</i> (<a href="http://onlinelibrary.wiley.com/doi/10.1111/jzs.12035/abstract">Journal of Zoological Systematics and Evolutionary Research <b>51</b>: 296-307</a>).</li>
<li><img src="/static/FinalOutputs/pics/2005140.jpg" alt="" /><strong>Lampreys</strong>: entire tree from figure 2.6 of Potter <i>et al.</i> (2015) <i>The Taxonomy, Phylogeny, and Distribution of Lampreys</i> (<a href="http://link.springer.com/chapter/10.1007/978-94-017-9306-3_2">Chapter 2 in Docker (ed.) Lampreys: Biology, Conservation and Control: Volume 1. Springer, Dordrecht</a>).</li>
</ul></li>
<li><strong>Sea urchins, starfish, acorn worms, etc</strong>:
<ul>
<li><b><i>Xenoturbella</i></b> is a controversial group of worms, whose position is much debated. In the current version of OneZoom they are placed according to Bourlat <i>et al.</i> (2006) <i>Deuterostome phylogeny reveals monophyletic chordates and the new phylum Xenoturbellida</i> (<a href="http://www.nature.com/nature/journal/v444/n7115/abs/nature05241.html">Nature <b>444</b>:85-88</a>). Note that a more recent study places them as a basal bilaterian phylum, sister to the acoelomorph flatworms - this study also identifies five species, four of which are new to science and are shown on the OneZoom tree: see Rouse <i>et al.</i> (2016) <i>New deep-sea species of Xenoturbella and the position of Xenacoelomorpha.</i> (<a href="http://www.nature.com/nature/journal/v530/n7588/abs/nature16545.html">Nature <b>530</b>: 94-97</a>).</li>
<li><img src="/static/FinalOutputs/pics/26780634.jpg" alt="" />The relationships between <strong>Echinozoa</strong> (sea urchins &amp; sea cucumbers), <strong>Ophiuroidea</strong> (brittle stars), <strong>Asteroidea</strong> (starfish), and <strong>Crinoidea</strong> (sea lilies) is resolved on the basis of Pisani <i>et al.</i> (2012) <i>Resolving phylogenetic signal from noise when divergence is rapid: A new look at the old problem of echinoderm class relationships</i> (<a href="doi:10.1016/j.ympev.2011.08.028">Molecular Phylogenetics and Evolution 62: 27-34</a>).</li>
</ul></li>
<li><img src="/static/FinalOutputs/pics/26781256.jpg" alt="" /><strong>Protosomes</strong>: apart from the arrow worms, the protostomes are now commonly believed to fall into two major groups, the <strong>Ecdysozoa</strong> and the another group which sometimes gets called <strong>Lophotrochozoa</strong> or <strong>Spiralia</strong>. Within these two groups:
<ul>
<li><img src="/static/FinalOutputs/pics/2004355.jpg" alt="" /><strong>Ecdysozoa</strong> (insects, and their wider relatives) now have a reasonably well established evolutionary tree. 
<ul>
<li><img src="/static/FinalOutputs/pics/26912835.jpg" alt="" />The <strong>Onychophora</strong> (velvet worms) are probably closest relatives of the <strong>arthropods</strong> (crustaceans, insects, spiders, centipedes, etc), as argued by Campbell <i>et al.</i> (2011) <i>MicroRNAs and phylogenomics resolve the relationships of Tardigrada and suggest that velvet worms are the sister group of Arthropoda</i> (<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3179045/">Proc Natl Acad Sci U S A. <b>108</b>: 15920-15924</a>). This is also suggested in the summary put together by Edgecombe <i>et al.</i> (2011) <i>Higher-level metazoan relationships: recent progress and remaining questions</i> <a href="http://dunnlab.org/assets/Edgecombe_etal_2011.pdf">Organisms, Diversity, and Evolution <b>11</b>:151-172</a>. Together with the tardigrades, these form the now well-established <strong>panarthropods</strong>.</li>
<li><img src="/static/FinalOutputs/pics/26792154.jpg" alt="" />Edgecombe <i>et al.</i> (2011) also group the parasitic <strong> Nematomorpha</strong> (horsehair worms) with <strong>Nematoda</strong> (nematode worms). Advice from taxonomists leads us to place this as the sister to the panarthropods</li>
<li><img src="/static/FinalOutputs/pics/13144676.jpg" alt="" /><strong>Scalidophora</strong> (a group of tiny obscure phyla) are left as the only remaining Ecdysozoans, hence the deepest Ecdysozoan split is between them and everything else. Note that this is arguable. For instance, Scalidophora are plaed as sister to nematomorpha/nematoda by Dunn <i>et al.</i> (2008) <i>Broad phylogenomic sampling improves resolution of the animal tree of life</i> <a href="http://www.nature.com/nature/journal/v452/n7188/abs/nature06614.html">Nature <b>452</b>: 745-9</a>.</li>
</ul></li>
<li><img src="/static/FinalOutputs/pics/5825170.jpg" alt="" /><strong>Spiralia</strong> (molluscs, segmented worms, and much more) &mdash; also known as Lophotrochozoa. The relationships between these animals is one of the trickiest in the entire tree. We mostly follow a recent review by Kocot (2016) <i>On 20 years of Lophotrochozoa</i> (<a href="">Organisms Diversity & Evolution <b>16</b>: 329-343</a>), and a slighly earlier paper by Laumer <i>et al.</i> (2015) <i>Spiralian Phylogeny Informs the Evolution of Microscopic Lineages</i> (<a href="http://www.cell.com/current-biology/abstract/S0960-9822(15)00795-2">Current Biology <b>25</b>: p2000-2006</a>).

<ul>
<li><img src="/static/FinalOutputs/pics/1995515.jpg" alt="" /><strong>Gnathifera</strong>, <strong>Gastrotricha</strong> and <strong>Platyhelminthes</strong>: Gnathifera is a collection of three very small, often wormy creatures, including the rotifers, which is sometimes placed with the flatworms (platyhelminths) and gastrotrichs in a group known as the <q>Platyzoa</q>. Although flatworms and gastrotrichs have been grouped together for almost a decade (e.g. see Dunn <i>et al.</i> <a href="http://www.nature.com/nature/journal/v452/n7188/abs/nature06614.html">Nature <b>452</b>: 745-9</a>), the broader idea of a coherent group of Platyzoans is increasingly thought to be wrong. We follow the alternative suggestion from Laumer <i>et al.</i> and Kocot that places Gnathifera as the earliest branching Spiralians, followed by the platyhelminth/gastrotrich grouping.</li>
<li><img src="/static/FinalOutputs/pics/27072192.jpg" alt="" /><strong>Bryozoans</strong> (moss animals), <strong>phoronids</strong> (horseshoe worms), and <strong>brachiopods</strong> are now often found joined together in molecular studies, forming a group known as the lophophorates (see Nesnidal <i>et al.</i>, (2013) <i>New phylogenomic data support the monophyly of Lophophorata and an Ectoproct-Phoronid clade and indicate that Polyzoa and Kryptrochozoa are caused by systematic bias.</i> <a href="http://www.biomedcentral.com/1471-2148/13/253/">BMC Evolutionary Biology <b>13</b>: 253</a>). Pleasingly, this is in line with older, morphological studies.</li>
<li><img src="/static/FinalOutputs/pics/5818107.jpg" alt="" /><strong>Entoprocta</strong> (~150 species of minute aquatic filter feeders) and <b>Cycliophora</b> (2 or 3 species of sac-like animals that live exclusively on lobster mouthparts) are often grouped together in molecular studies. Kocot very tentatively suggests that this group could be <em>within</em> the lophophorates, as sister to the <strong>bryozoans</strong>, in line with morphological studies that suggest these three phyla form a higher level group called the <strong>Polyzoa</strong>. Until a better suggestion comes along, we follow this (rather unsubstantiated) suggestion.</li>
<li><img src="/static/FinalOutputs/pics/27480684.jpg" alt="" />The <strong>ribbon worms</strong>, <strong>segmented worms</strong> (annelids, e.g. earthworms) and <strong>molluscs</strong> are grouped by Edgecombe <i>et al.</i> with the lophophorates, in a larger band called Trochozoa. This grouping is also backed by Laumer <i>et al.</i> and Kocot. For the relationships between these four groups, we follow Laumer <i>et al.</i> in placing the basal divergence between molluscs and the rest, followed by a divergence between annelids and the remaining two. This particular arrangement is, however, extremely tentative.</li>
<li><img src="/static/FinalOutputs/pics/26785798.jpg" alt="" />Within the <strong>segmented worms</strong>, relationships have been taken from Weigert &amp; Bleidorn (2016) <i>Current status of annelid phylogeny</i> (<a href="http://link.springer.com/article/10.1007/s13127-016-0265-7">Organisms Diversity & Evolution <b>16</b>: 345-362</a> figure 2) rather than relying on the the OpenTree, which has particular problems with the placement of Myzostomida. Note that the OpenTree currently excludes the groups Cirratuliformia, Dinophilidae, Glyceriformia, Nerillidae, Polygordiidae, Protodrilidae, and Protodriloidae, so these are currently absent from the OneZoom tree.</li>
</ul></li>
<li><img src="/static/FinalOutputs/pics/16890934.jpg" alt="" />Finally, the planktonic arrow worms (<strong>chaetognaths</strong>) are a perennial bugbear, but are considered by nearly all biologists to be protostomes, for example by Perez <i>et al.</i> (2014) <i>The Chaetognatha: An anarchistic taxon between Protostomia and Deuterostomia</i> (<a href="https://books.google.co.uk/books?id=T67pBQAAQBAJ&amp;pg=PA49">in: W&auml;gele &amp; Bartolomaeus (eds.) Deep Metazoan Phylogeny: The Backbone of the Tree of Life</a>). As for the relationship between chaetognaths and the other protosomes, they have been variously placed as early-diverging ecdysozoans, spiralians, or at the base of all the protostomes &mdash; we choose the last of these after consultation with taxonomists and on the basis of a paper by Shen <i>et al.</i> (2015) <i>Phylomitogenomic analyses strongly support the sister relationship of the Chaetognatha and Protostomia</i> (<a href="http://onlinelibrary.wiley.com/doi/10.1111/zsc.12140/abstract">Zoologica Scripta:<b>45</b>: 187-199</a>).</li>
</ul></li>
<li><img src="/static/FinalOutputs/pics/26793885.jpg" alt="" /><strong>Ctenophores</strong> (comb jellies): arrangement of orders and families from Poder <i>et al.</i> (2001) <i>A molecular phylogenetic framework for the phylum Ctenophora using 18S rRNA genes</i> (<a href="http://www.sciencedirect.com/science/article/pii/S105579030191036X">Molecular Phylogenetics and Evolution <b>21</b>: 218-230</a>). On taxonomic advice, we do <em>not</em> follow the recent suggestion that ctenophores are the sister group to all other animals (including humans).</li>
<li><strong>Cnidarians</strong>:
<ul>
<li><img src="/static/FinalOutputs/pics/1995506.jpg" alt="" /> <strong>Myxozoa</strong>: placed as basal cnidarians according to Nesnidal <i>et al.</i> (2013) <i>Agent of Whirling Disease Meets Orphan Worm: Phylogenomic Analyses Firmly Place Myxozoa in Cnidaria</i> (<a href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0054576">PLoS ONE 8: e54576</a>).</li>
<li><img src="/static/FinalOutputs/pics/26933308.jpg" alt="" /><strong>Anthozoa &amp; Medusozoa</strong> Families placed according to Collins (2009) <i>Recent Insights into Cnidarian Phylogeny</i> (<a href="http://decapoda.nhm.org/pdfs/31459/31459-001.pdf">in: Lang, M.A. et al. (2009). Proceedings of the Smithsonian Marine Science Symposium. Smithsonian Contributions to the Marine Sciences <b>38</b>: 139-149</a>).</li>
</ul></li>
<li><img src="/static/FinalOutputs/pics/31923634.jpg" alt="" /><strong>Holomycota</strong> (fungi etc.): The OpenTree has been used to fill out all the high-level fungal groups: Discicristoidea (placed at the base), Neocallimastigomycetes, Chytridiomycetes, Blastocladiomycota, Mucoromycotina, Ascomycota, Basidiomycota, Glomeromycota, Entomophthoromycota, Kickxellomycotina, Zoopagomycotina.
<ul>
<li><img src="/static/FinalOutputs/pics/31800338.jpg" alt="" /><strong>Cryptomycota and Microsporidia</strong>: these group together, according to James <i>et al.</i> (2013) <i>Shared signatures of parasitism and phylogenomics unite Cryptomycota and microsporidia</i> (<a href="http://www.ncbi.nlm.nih.gov/pubmed/23932404">Current Biology <b>23</b>1548-53</a>), together with the <strong>Aphelidea</strong> (see Karpov <i>et al.</i> (2014) <i>Morphology, phylogeny, and ecology of the aphelids (Aphelidea, Opisthokonta) and proposal for the new superphylum Opisthosporidia</i> (<a href="http://dx.doi.org/10.3389/fmicb.2014.00112">Frontiers in Microbiology <b>5</b>:112</a>))</li>
<li> <img src="/static/FinalOutputs/pics/26827086.jpg" alt="" /> Microsporidia and allies are the deepest branching group in the fungi, followed by the <strong>Chytridiomycota</strong>, according to Capella-Guti&eacute;rrez <i>et al.</i> (2012) <i>Phylogenomics supports microsporidia as the earliest diverging clade of sequenced fungi</i> (<a href="http://www.biomedcentral.com/1741-7007/10/47">BMC Biology <b>10</b>:47</a>).</li>
<li> <img src="/static/FinalOutputs/pics/3603008.jpg" alt="" /> <strong>Entomophthoromycota</strong> are placed as an outgroup to Kickxellomycotina + Zoopagomycotina as a result of Gryganskyi <i>et al.</i> (2012) <i>Molecular phylogeny of the Entomophthoromycota</i> (<a href="http://www.ncbi.nlm.nih.gov/pubmed/22877646">Molecular Phylogenetics and Evolution <b>65</b>:682-94</a>).</li>
<li>Other fungal groups (previously lumped into &#39;<strong>zygomycota</strong>&#39; agree with the arrangement found in Porter <i>et al</i> (2011) <i>Molecular phylogeny of the Blastocladiomycota (Fungi) based on nuclear ribosomal DNA</i> (<a href="http://www.ncbi.nlm.nih.gov/pubmed/21530920">Fungal Biology <b>115</b>: 381-92</a>). Also see the summary by Stajich <i>et al.</i> in <a href="http://www.cell.com/current-biology/abstract/S0960-9822(09)01382-7">Current Biology <b>19</b>: pR840-R845</a>.</li>
</ul></li>
<li><p><strong>Apusozoa</strong>: currently of uncertain reliability. See Cavalier-Smith &amp; Chao (2010) <i>Phylogeny and evolution of apusomonadida (protozoa: apusozoa): new genera and species</i> (<a href="http://www.sciencedirect.com/science/article/pii/S1434461010000246">Protist <b>161</b>:549-76</a>).</p></li>
<li><img src="/static/FinalOutputs/pics/26876943.jpg" alt="" /><strong>Plants, brown algae, etc</strong> (bikonts): higher-level arrangements from Burki (2014) <i>The Eukaryotic Tree of Life from a Global Phylogenomic Perspective</i> (<a href="http://cshperspectives.cshlp.org/content/6/5/a016147">Cold Spring Harbour Perspectives in Biology <b>6</b>: a016147</a>), with supplementary positioning of <img src="/static/FinalOutputs/pics/26885891.jpg" alt="" />Cryptophytes, <i>Collodictyon</i>, malawimonads,  centrohelids and haptophytes from personal communication with Fabien Burki (see also Burki <i>et al.</i> (2016) <i>Untangling the early diversification of eukaryotes: a phylogenomic study of the evolutionary origins of Centrohelida, Haptophyta and Cryptista</i> (<a href="http://rspb.royalsocietypublishing.org/content/283/1823/20152802">Proceedings of the Royal Society B <b>283</b>: 20152802</a>)). <img src="/static/FinalOutputs/pics/3601378.jpg" alt="" />Collodictyonidae and Rigifilida are grouped together as per Adl <i>et al.</i> (2012) <i>The revised classification of eukaryotes</i> (<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3483872/">Journal of Eukaryotic Microbiology <b>59</b>: 429-493</a>). The following orders and groups of bikonts have then been filled out to species level from the OpenTree: Excavata, Haptophyta, Centrohelida, Picozoa, Alveolata, Stramenopiles, Rhizaria, Cryptophyceae, Glaucophyta, Rhodophyceae, Chloroplastida.</li>

<li> <img src="/static/FinalOutputs/pics/26400599.jpg" alt="" /> <p><strong>Archaea</strong> are taken to contain the Eukaryotes. This is the &quot;2 Domain&quot; hypothesis as described in Williams &amp; Embley (2014) <i>Archaeal &quot;Dark Matter&quot; and the Origin of Eukaryotes</i> (<a href="http://gbe.oxfordjournals.org/content/6/3/474.full">Genome Biology and Evolution <b>6</b>: 474-481</a>). The detailed arrangements are mostly taken from the OpenTree but with:</p>
<ul>
<li><strong>Lokiarchaeota</strong>: closest to Eukaryotes, as per Spang <i>et al.</i> (2015) <i>Complex archaea that bridge the gap between prokaryotes and eukaryotes</i> (<a href="http://www.nature.com/nature/journal/v521/n7551/full/nature14447.html">Nature <b>521</b>: 173-179</a>).</li>
<li><img src="/static/FinalOutputs/pics/26400600.jpg" alt="" />The <q><strong>TACK</strong></q> archeal group are considered next closest relatives to the eukaryotes, as discussed by Guy &amp; Ettema (2011) <i>The archaeal &#39;TACK&#39; superphylum and the origin of eukaryotes</i> (<a href="http://www.ncbi.nlm.nih.gov/pubmed/22018741">Trends in Microbiology <b>19</b>: 580-7</a>). Note that these are listed as [Crenarchaeota, Thermococci, Thaumarchaeota, Thermoplasmata, &amp; Korarchaeota] in the OpenTree version 7.</li>
<li>The <strong>dPANN</strong> group united as the next closest relatives to Eukaryotes (these are Nanoarchaeota, Diapherotrites, Aenigmarchaeota, Nanohaloarchaeota, Micrarchaeum, &amp; Parvarchaeum) as seen in Rinke <i>et al.</i> (2013) <i>Insights into the phylogeny and coding potential of microbial dark matter</i> (<a href="http://www.nature.com/nature/journal/v499/n7459/full/nature12352.html">Nature <b>499</b>: 431-437</a>). </li>
<li>The remaining groups have been taken as &quot;<strong>Euryarchaeota</strong>&quot;, and placed as the outgroup to everything else.</li>
</ul></li>
</ul>

<h4>Dates on the OneZoom tree</h4>
<p>Dates are taken from <a href="http://www.ancestorstale.net">The Ancestor&#39;s Tale (2nd edition)</a>, which provides a detailed explanation in the endnotes about the dating techniques used.</p>
</body></html>
